pages/index.js

```diff
@@ -1,20 +1,19 @@
 import React from 'react';
 import Layout from '../components/Layout';
 import EventList from '../components/EventList';
-import EVENTS from '../fixtures/events'
+import { fixtureApi } from '../services';
```

Comment:
all the exports from `services/index.js` are named exports -- the `default` you see in there is short-hand for importing the default from `./fixturesApi` and then does a named export as `fixturesApi`